src/test/features/2-ExerciseOptions.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Exercise Options
+
+    As an Option Holder,
+    I want to exercise options for a pair of ERC20 assets,
+    so that I can pursue advanced trading strategies on Ethereum.
+
+    Background: XYZ
```

Comment:
do we want to populate these scenarios?

Reply:
We can do more detail here after freeze of the core contract.